test/core/security/rbac_translator_test.cc

```diff
@@ -107,8 +107,7 @@ TEST(GenerateRbacPoliciesTest, MissingDenyRules) {
       "}";
   auto rbac_policies = GenerateRbacPolicies(authz_policy);
   ASSERT_TRUE(rbac_policies.ok());
-  EXPECT_EQ(rbac_policies.value().size(), 1);
-  EXPECT_EQ(rbac_policies.value().at(0).action, Rbac::Action::kAllow);
+  EXPECT_FALSE(rbac_policies.value().deny_policy.has_value());
```

Comment:
This can say `rbac_policies->deny_policy` instead of `rbac_policies.value().deny_policy`.

Reply:
Done